src/Common/src/Interop/Windows/mincore/Interop.ConsoleScreenBufferInfo.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [StructLayoutAttribute(LayoutKind.Sequential)]
+        internal struct CONSOLE_SCREEN_BUFFER_INFO
```

Comment:
It does but CONSOLE_SCREEN_BUFFER_INFO is the exchange type of the pinvoke (it is the return type) which gets used by multiple methods in ConsolePAL. I think it is achievable to use a different exchange type like the color attributes (short) which is what most of those methods are using but I chose this because it required minimum code change. Let me know if you feel strongly about it :).